app/Models/Concerns/PostHasReminder.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace App\Models\Concerns;
+
+use App\Models\Reminder;
+use App\Observers\Remindable;
+use App\Observers\RemindablePost;
+use Illuminate\Support\Arr;
+
+/**
+ * Trait PostHasReminder
+ *
+ * @property Reminder $calendarReminder
+ * @property Reminder|null $calendarDate
+ * @property int|null $calendar_id
+ * @property int|null $calendar_year
+ * @property int|null $calendar_month
+ * @property int|null $calendar_day
+ */
+trait PostHasReminder
```

Comment:
This is basically a copy-pasted HasReminder but skipping `->entity` and changing the observer. Can this be merged into a single trait instead?